src/pages/knowledge-base/[slug].page.tsx

```diff
@@ -38,6 +39,26 @@ const Post = ({ post, recents, categories }: Props) => {
     }
   }
 
+  const [isAuthorListExpanded, setIsAuthorListExpanded] = useState<boolean>(false)
+  const toggleAuthorListExpanded = () => {
+    setIsAuthorListExpanded(!isAuthorListExpanded)
+  }
+  const renderExpandedAuthors = () => {
+    if (!isAuthorListExpanded) {
+      return null
+    }
+    return (
+      <div className={styles.expandedAuthors}>
+        {[...post.authors].map(({ name, avatar }) => (
+          <div className={styles.expandedAuthorItem} key={`expanded-author-item-${name}`}>
+            <img src={avatar} className={styles.expandedAuthorAvatar} alt="avatar" />
+            <div>{name}</div>
+          </div>
+        ))}
+      </div>
+    )
+  }
```

Comment:
It can be defined as a component, shared between `/knowledge-base` page and `/knowledge/[slug]` page

Reply:
updated